precompiles/staking/StakingI.sol

```diff
@@ -238,6 +238,24 @@ interface StakingI is authorization.AuthorizationI {
             PageResponse calldata pageResponse
         );
 
+
+    // Restakes the rewards earned by the delegator to the validator.
+    // @param delegatorAddress The address of the delegator.
+    // @param validatorAddress The address of the validator.
+    // @return success Whether or not the restake was successful.
+    function restake(
+        address delegatorAddress,
+        address validatorAddress
```

Comment:
FYI, we will need to add the Bech32 Extension if we want to make this an address